### PR TITLE
home category slug  & naming comvention change

### PR DESCRIPTION
**Pull request by mahendra-lexeslab**
hi admin ,
 please review this branch

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 PORT=3000
-DATABASE_URL="mongodb://13.233.70.196:27017"
+# DATABASE_URL="mongodb://13.233.70.196:27017"
```
Comment (admin-lexes) [inline .env:2]: remote server

File: controllers/categoryController.js [hunk 2]
```diff
@@ -1,50 +1,115 @@
...
+                if (category) {
+                    await category.save()
+                    res.status(201).json({ category: category, message: "new category Created" });
+                }
+
+            } else {
+                res.status(404).json({ message: "please provide right data catName , type  ,refrenceId" });
+            }
         } catch (error) {
             console.log(error)
```
Comment (admin-lexes) [inline controllers/categoryController.js:66]: exception to hadnle carefully

**Review by admin-lexes — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (mahendra-lexeslab) [reply .env:2]: add remote  server

File: controllers/categoryController.js (hunk 2 above)
Comment (mahendra-lexeslab) [reply controllers/categoryController.js:66]: add exception in code.

**Pull request merged**